src/pages/HotelListPage/components/OptionBar.tsx

```diff
@@ -0,0 +1,28 @@
+import { useState } from 'react';
+import {
+  optionButton,
+  optionWrapper,
+  activeBorderButton,
+  activeText,
+} from './OptionBar.style';
+
+const OptionBar = () => {
+  const [isLimitedSaleActive, setIsLimitedSaleActive] = useState(false);
+
+  return (
+    <div css={optionWrapper}>
+      <button css={optionButton}>
+        정렬 &nbsp;|&nbsp; <span css={activeText}>추천순</span>
+      </button>
```

Comment:
아주아주 사소한 부분이지만 .. 개인적으로 `&nbsp` 사용하는 것보단 텍스트를 모두 span 태그로 나누어서 작성하는게 좀 더 직관적이지 않을까 생각해요!

```suggestion
      <button css={optionButton}>
        <span>정렬</span>
        <span>|</span>
        <span css={activeText} >추천순</span>
      </button>
```

이렇게 나누고, button 스타일에 flex 설정 후 gap을 주는 방식으로 구현하면, 공백 크기를 조정할 때도 더 수월하지 않을까 ..🤤

Reply:
백번 맞는말입니다. 귀찮아서 그냥 냅뒀었네요 ,, ㅎㅎ